scripts/validate-examples.sh

```
@@ -0,0 +1,112 @@
+    exit 1
+fi
+
+# Install provider to plugin directory
+printf "  ${CYAN}→${RESET} Installing provider to plugin directory\n"
+mkdir -p "$PROVIDER_DIR"
+cp "$PROVIDER_BIN" "$PROVIDER_DIR/"
+chmod +x "$PROVIDER_DIR/terraform-provider-n8n"
+
+# Setup dev overrides config
+TF_CLI_CONFIG_FILE=$(mktemp)
+trap "rm -f $TF_CLI_CONFIG_FILE" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)